src/Service/Handler.php

```diff
@@ -102,55 +170,88 @@ public function handle(EntityInterface $node, $action) {
     }
   }
 
+  /**
+   * Get event data.
+   *
+   * @param \Drupal\node\NodeInterface $node
+   *   The node to get event data for.
+   *
+   * @return array<string, mixed>
+   *   The event data.
+   */
   private function getEventData(NodeInterface $node) {
     $valueHandler = new ValueHandler();
     $transformer = new ObjectTransformer($valueHandler);
     $config = $this->getMapping($node);
 
     $data = $transformer->transformObject($node, $config);
 
-    $url = Url::fromRoute('entity.node.canonical', ['node' => $node->id()], ['absolute' => true]);
+    $url = Url::fromRoute('entity.node.canonical', ['node' => $node->id()], ['absolute' => TRUE]);
```

Comment:
Should we use that weird `$url->toString(TRUE)->getGeneratedUrl()` stuff when setting `$data['url']` in the following statement (cf. https://www.drupal.org/project/drupal/issues/2735575)?

Reply:
Changed